test/module/irohad/pending_txs_storage/pending_txs_storage_test.cpp

```diff
@@ -57,8 +61,47 @@ class PendingTxsStorageFixture : public ::testing::Test {
     return addSignatures(
         makeTestBatch(txBuilder(2, getUniqueTime(), 2, "alice@iroha"),
                       txBuilder(2, getUniqueTime(), 2, "bob@iroha")),
-        0,
-        makeSignature("1", "pub_key_1"));
+        0, makeSignature("1", "pub_key_1"));
+  }
+
+  void checkResponse(const Response &actual, const Response &expected) {
+    EXPECT_EQ(actual.transactions.size(), expected.transactions.size());
+    // generally it's illegal way to verify the correctness.
+    // here we can do it because the order is preserved by batch meta and
+    // there are no transactions non-related to requested account
+    for (auto i = 0u; i < expected.transactions.size(); ++i) {
+      EXPECT_EQ(*actual.transactions[i], *expected.transactions[i]);
+    }
+    EXPECT_EQ(actual.all_transactions_size, expected.all_transactions_size);
+    if (expected.next_batch_info) {
+      EXPECT_TRUE(actual.next_batch_info);
+      EXPECT_EQ(actual.next_batch_info->first_tx_hash,
+                expected.next_batch_info->first_tx_hash);
+      EXPECT_EQ(actual.next_batch_info->batch_size,
+                expected.next_batch_info->batch_size);
+    } else {
+      EXPECT_FALSE(actual.next_batch_info);
+    }
+  }
+
+  void validateResponse(
+      const iroha::expected::Result<Response, ErrorCode> &pending,
+      const Response &expected) {
+    try {
+      checkResponse(pending.assumeValue(), expected);
+    } catch (iroha::expected::ResultException &e) {
+      FAIL() << "Error validating results: " << e.what();
+    }
```

Comment:
we have utils for `Result`-related tests available with
```c++
#include "framework/result_gtest_checkers.hpp"
```
```suggestion
    IROHA_ASSERT_RESULT_VALUE(pending);
    checkResponse(pending.assumeValue(), expected);
```